tst/Carnival/Unit/Data/Hooks/ExampleHook.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Tests\Carnival\Unit\Data\Hooks;
```

Comment:
Since this is a data class, it doesn't really fall under the "Unit" category. Perhaps `Tests\Carnival\Data\Hooks`?

Reply:
Also, this would be an example event, no? Not an example "hook"? Though, per above comment, the "Event" name is also subject to change